test/fixtures/typescript/vulnerable.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs-extra')
+var stdin = process.openStdin()
+
+function bildngSQLQuery () {
+  const userId = 1
+  const query = `SELECT * FROM users WHERE id = ${userId}`
+  return query
+}
+
+function getTemplateGreeting (name) {
+  let classicString = `Hello ${name}`
+  return eval('`' + classicString + '`') // eslint-disable-line
```

Comment:
Is the eslint comment still useful?

Reply:
It's useful because we are using [JavaScript standard](https://standardjs.com/) and [Jest]https://jestjs.io/ to enforce style guidelines.
Before we run our unit tests JavaScript standard runs through jest to make sure that the standard JavaScript style is followed. 

In this case, JavaScript Standart reports the use of eval function as a problem and as a result, the Travis CI build will fail.
in order to escape that I use "eslint-disable-line" comment because under the hood JavaScript standard uses ESlint.
As described here: https://standardjs.com/#how-do-i-hide-a-certain-warning

Reply:
Thanks for explaining.